packages/homepage/content/articles/codesandbox-ci-embeds/index.md

```diff
@@ -0,0 +1,102 @@
+---
+banner: ./images/oldvsnew.png
+slug: codesandbox-ci-embeds
+authors: ['Ives van Hoorne']
+photo: https://avatars0.githubusercontent.com/u/587016?s=460&v=4
+title: Announcing CodeSandbox CI and all-new Embeds
+description:
+  Today we're pleased to launch CodeSandbox CI - A free continuous integration
+  service purpose-built for open source library maintainers, along with a new
+  look for embeds.
+date: 2019-11-13
+---
+
+## CodeSandbox CI
+
+Maintaining open source projects, while enormously fulfilling, can be
+time-consuming and stressful. After a busy Hacktober, we know from first-hand
+experience what it's like, and we wanted to help library maintainers make the
+best use of their time. Today we're pleased to launch CodeSandbox CI: A free
+continuous integration service purpose-built for open source library
+maintainers.
+
+<iframe width="100%" height="500" src="https://www.youtube.com/embed/k_aR_4FeKlc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
This can be handled by [`gatsby-remark-embedder`](https://github.com/MichaelDeBoey/gatsby-remark-embedder) too

```suggestion
https://youtu.be/k_aR_4FeKlc
```